src/Conversion/ONNXToKrnl/RNN/RNNBase.cpp

```diff
@@ -247,12 +247,6 @@ Value applyActivation(OpBuilder &rewriter, Location loc,
   bool isScalar = !operand.getType().isa<ShapedType>();
```

Comment:
Out of curiosity, before you were including vectors in scalar. So you are intending to not include them here? I think its ok since we do not vectorize before ONNX to KRNL, but I just wonder why it was an issue in the above code but not here.

Reply:
@AlexandreEichenberger Yes, you are right, they are redundant. I just removed them. Thanks!